Automattic/woocommerce-payments
# Redirect unavailable Payments > Overview to the Payments > Connect page

Fixes #6011 

#### Changes proposed in this Pull Request

When a merchant is directed to the Payments > Overview page (e.g. hitting return in the Stripe KYC) and that WC page is not registered, we redirect them to the Payments > Connect page, if it is registered. This way we avoid a "Not allowed" semi-blank page.

I've also made a small improvement in the JS of the Payments > Overview page to display transient-based error messages also.

#### Testing instructions

* Checkout the PR branch
* Go to WCPay Dev Tools and click on the Re-onboard link
* Once in the Stripe KYC, click the return link
* You should be redirected to the Payment > Connect page.

-------------------


**Post merge**
